src/frontend/src/components/NFTMetadataFetcher.css

```
@@ -0,0 +1,48 @@
+    border-radius: 4px;
+  }
+  .error {
+    color: red;
+    margin: 10px 0;
+  }
+  form {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    margin-top: 10px;
+    text-color: black;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)